test/e2e/components/vl-infoblock.js

```diff
@@ -0,0 +1,63 @@
+const { VlElement } = require('vl-ui-core');
+const { VlIcon } = require('vl-ui-icon');
+
+class VlInfoblock extends VlElement {
+    constructor(driver, selector) {
+        super(driver, selector);
+    }
+
+    async getText() {
+        const element = await this._getContent();
+        return this.driver.executeScript('return arguments[0].innerText', element);
+    }
```

Comment:
Kan de getText van VlElement (lees: Selenium webdriver) niet gebruikt worden?

Reply:
Die neemt niet altijd mooi de tekst als dat in een nested element zit.